dev/com.ibm.ws.cdi.visibility_fat/.classpath

```diff
@@ -26,7 +26,7 @@
 	<classpathentry kind="src" path="test-applications/maskedClassLib.jar/src"/>
 	<classpathentry kind="src" path="test-applications/visTestNonLib.jar/src"/>
 	<classpathentry kind="src" path="test-applications/sharedLibrary.jar/src"/>
-	<classpathentry kind="src" path="test-applications/TestValidatorInJar.ear/src"/>
+	<classpathentry kind="src" path="test-applications/TestValidatorInJar.ear/resources"/>
```

Comment:
Since there is no java source in this directory, I didn't think we included it in .classpath.  At least other projects that have 'resources' directories with just XML have not done this.